### PR TITLE
Remove extra space in Rerun build with failed targets link

### PR DESCRIPTION
**Pull request by Nuel-Eneji**
### Input trim() to check trailing whitespace
I added the trim() to remove whitespace before and after the string

this draft PR attempts to fix issue #4063 

singed-off-by: Nuel Eneji <enejiemmanueladie@gmail.com>

### CODE REVIEW

**Comment by Nuel-Eneji** [comment]
Hello, @smlambert I hope everything is good with you.
I just made a draft pull request as directed and used the trim() function where it was thought to be suitable.
I would like to take part in the PR review process.
Thanks

**Comment by Nuel-Eneji** [comment]
Hello @smlambert I'm yet to get a response from you.
I'll like to know the progress

**Comment by llxia** [comment]
@Nuel-Eneji could you please remove the changes under .github/ISSUE_TEMPLATE? I believe they are not related to the fix of https://github.com/adoptium/aqa-tests/issues/4063

**Comment by Nuel-Eneji** [comment]
Hello @llxia sorry for the inconvinience, I've reverted the changes as made.
Thanks for the review

File: buildenv/jenkins/JenkinsfileBase [hunk 1]
```diff
@@ -1073,7 +1073,7 @@ def addFailedTestsGrinderLink(paths=""){
 		def failedTestList = ""
 		List<String> buildPaths = paths.split(',')
 		for (def buildPath: buildPaths) {
-			def tapFiles = findFiles(glob: "${buildPath}**/*.tap")
+			def tapFiles = findFiles(glob: "${buildPath.trim()}**/*.tap")
```
[the remark this answers is not in the document]
Comment (Nuel-Eneji) [reply buildenv/jenkins/JenkinsfileBase:1076]: Thanks @llxia I'll get on with that now

**Comment by Nuel-Eneji** [comment]
Hello @smlambert @llxia  I moved the `trim()` function to L1084 as suggested. Please can you help me explain why you suggested the function be placed on that line as I'm not to sure of what type of data passes through the functions on that line 
Thanks

File: buildenv/jenkins/JenkinsfileBase (hunk 1 above)
[the remark this answers is not in the document]
Comment (Nuel-Eneji) [reply buildenv/jenkins/JenkinsfileBase]: Hello @llxia Thanks for the review. Please i need further guidiance on this 
Thanks

**Review by llxia — APPROVED** [review]
LGTM

**Review by smlambert — APPROVED** [review]

**Pull request merged**